core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -527,29 +544,6 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
       pendingReplayTasksCount.addAndGet(tasks.size)
```

Comment:
Is this number still valid? Now all submitted tasks in the queue are sum up to the count?

Reply:
You are right, I will increment the pending tasks count when a task begin running.